pkg/test/kube/dump.go

```diff
@@ -72,6 +72,44 @@ func DumpDeployments(ctx resource.Context, workDir, namespace string) {
 			})
 		}
 	}
+	errG.Wait()
+}
+
+func DumpWebhooks(ctx resource.Context, workDir string) {
+	errG := multierror.Group{}
+	for _, cluster := range ctx.AllClusters().Kube() {
+		mwhs, err := cluster.AdmissionregistrationV1().MutatingWebhookConfigurations().List(context.TODO(), metav1.ListOptions{})
+		if err != nil {
+			scopes.Framework.Warnf("Error getting mutating webhook configurations: %v", err)
+			return
+		}
+		for _, mwh := range mwhs.Items {
+			mwh := mwh
+			errG.Go(func() error {
```

Comment:
this seems a bit excessive, I do not think writing to disc in parallel will help much. Plus should only be a few items